pkg/aws/validations/iam_helpers.go

```diff
@@ -62,3 +66,25 @@ func IamResourceHasTag(iamTags []iamtypes.Tag, tagKey string, tagValue string) b
 
 	return false
 }
+
+func IamRoleArnsValidator(cluster *cmv1.Cluster) error {
+	iamRoleArns := map[string]struct{}{}
+	accountRolesMap := GetAccountRolesArnsMap(cluster)
+	operatorRolesMap := GetOperatorRolesArnsMap(cluster)
```

Comment:
can we use maps.Copy() to merge the two maps here? Then later we can use only one loop to check the duplicate arns, to avoid duplicate code.